### ISSUE TITLE
Error when Formatting(#if swift(>=X.Y))

### ISSUE THREAD

**Issue by RafaelPlantard** [Jintin/Swimat#127]
It's formatting this:
`#if swift(>=3)`
To this:
`#if swift( >= 3)`
Some spaces are unnecessary.

**Comment by Jintin** [comment]
Are you using extension version, if so please update and try again.

**Comment by RafaelPlantard** [comment]
Yep, I'm using the extension version... Thanks 😆

**Comment by SeanRobinson159** [comment]
I found another formatting issue.
At the ending, or start of the next closure (if more than one) is formatting incorrectly

<img width="626" alt="screen shot 2016-10-05 at 8 41 02 am" src="https://cloud.githubusercontent.com/assets/7306385/19117905/e81be486-8ad7-11e6-8f7b-13c7aa25aa61.png">

<img width="538" alt="screen shot 2016-10-05 at 8 42 35 am" src="https://cloud.githubusercontent.com/assets/7306385/19117912/ee0f072e-8ad7-11e6-8e8d-1a18d9564d0f.png">

I am using the latest version of the Extension branch, compiled and added manually to Xcode Version 8.0 (8A218a).

**Comment by Jintin** [comment]
Thanks for report @SeanRobinson159, please update and try again.

**Comment by SeanRobinson159** [comment]
I tried, no change. The last commit was a couple days ago though. Did you forget to push?

**Comment by Jintin** [comment]
Sorry, I have a long day.
Please try again.

**Comment by SeanRobinson159** [comment]
No problem, I'm just thankful you are there to work on it! It works great now! Thank you so much. I'd love to help with the project, but it is a little hard to follow. If I figure it out, I'll just submit pull requests if I find more issues that I can fix.
